Katello/katello
# Fixes #29228 - Ignore 404 on delete

To test:
1. Create a repo and sync
2. curl --cert /etc/pki/katello/certs/pulp-client.crt --key /etc/pki/katello/private/pulp-client.key https://`hostname`/pulp/api/v3/repositories/file/file/  to get the **repo_href.**
3. curl -X DELETE --cert /etc/pki/katello/certs/pulp-client.crt --key /etc/pki/katello/private/pulp-client.key https://`hostname`/pulp/api/v3/repositories/file/file/**repo_href**/
4. Delete the repo in katello UI. It should succeed.
